draft-ietf-httpbis-message-signatures.md

```diff
@@ -1904,6 +1907,12 @@ To counter this, an application needs to validate the content of the fields cove
 
 Multiple applications and protocols could apply HTTP signatures on the same message simultaneously. A naive verifier could become confused in processing multiple signatures, either accepting or rejecting a message based on an unrelated or irrelevant signature. In order to help an application select which signatures apply to its own processing, the application can declare a specific value for the `context` signature parameter. For example, a signature targeting an application gateway could require `context="app-gateway"` as part of the signature parameters for that application.
 
+## Message Component Context {#security-message-component-context}
```

Comment:
The section title is very general, but most of the text is about one single use case (a reverse proxy).

Reply:
You're totally right -- but @richanna is working on reviewing and expanding this right now. :) I just needed to git it up on GitHub so she could take a pass at it, too.